packages/admin/cms-admin/src/contentScope/Provider.tsx

```diff
@@ -125,9 +125,16 @@ export interface ContentScopeProviderProps {
     values?: ContentScopeValues;
     children: (p: { match: match<NonNullRecord<ContentScope>> }) => ReactNode;
     location?: ContentScopeLocation;
```

Comment:
Following exception:

```        throw new Error("ContentScopeProvider: values and defaultValue couldn't be evaluated from currentUser and must therefore be provided.");``` will end in a blank page, and a console error message.

We should also show something appropriate here with a suiteable error message.

Reply:
that's right, that's nothing what will happen for one special user that will happen everytime during development.

Than lets keep this like it is